### DISCUSSION TITLE
Feature request: publish @fortawesome/fontawesome-svg-core package with untranspiled javascript

### DISCUSSION THREAD

**Discussion by blikblum** [FortAwesome/Font-Awesome#20959] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently `@fortawesome/fontawesome-svg-core` is published with two js files, one with umd loader and other with es module, both with code transpiled by babel down to es5.

**Describe the solution you'd like**
Publish a third js file with modern, untranspiled code so those that supports only evergreen browsers can benefit from smaller, faster code.

For backward compatibility, this new file should not be linked to main or module package entry. To use it those that need should import directly

**Describe alternatives you've considered**
None.

**Additional context**
Many libraries are publishing modern, untranspiled js code these days

**Feature request checklist**
- [x] This is a single feature (i.e. not a re-write of all of Font Awesome)
- [x] The title starts with "Feature request: " and is followed by a clear feature name (Ex: `Feature request: moar cowbell`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome-Pro/issues) and to the best of my knowledge this is not a duplicate

**Comment by waterfoul** [comment]
I would also like to see this. We use babel as part of our build process and importing the index.es.js file results in a non-trivial amount of duplicate code (the helpers).
